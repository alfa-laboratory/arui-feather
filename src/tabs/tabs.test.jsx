/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from 'react';
import { mount } from 'enzyme';

import Tabs from './tabs';

describe('tabs', () => {
    it('should render without problems', () => {
        let tabs = mount(<Tabs />);

        expect(tabs).toMatchSnapshot();
    });

    it('should set scrollable className when scrollable=true without problems', () => {
        let tabs = mount(<Tabs />);

        expect(tabs.getDOMNode().className).toContain('tabs_scrollable');
    });
});
