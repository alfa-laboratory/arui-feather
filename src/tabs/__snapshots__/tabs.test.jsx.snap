// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tabs should render without problems 1`] = `
<Tabs
  scrollable={true}
>
  <div
    className="tabs tabs_scrollable tabs_theme_alfa-on-white"
  >
    <div
      className="tabs__panel"
    >
      <div
        className="tabs__content"
      />
    </div>
  </div>
</Tabs>
`;
